<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlockResize</class>
 <widget class="QWidget" name="BlockResize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>126</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="AutoSelectSpinBox" name="videoWidthButton">
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string> px</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>1920</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="AutoSelectSpinBox" name="videoHeightButton">
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string> px</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>1080</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="modeBox">
     <item>
      <property name="text">
       <string>Letterbox</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Crop</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stretch</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="DataComboBox" name="algorithmBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Algorithm</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionHD1080">
   <property name="text">
    <string>HD ( 1920x1080 - 1.78:1 )</string>
   </property>
  </action>
  <action name="actionHD720">
   <property name="text">
    <string>HD ( 1280x720 - 1.78:1 )</string>
   </property>
  </action>
  <action name="actionKeepRatio">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Keep ratio</string>
   </property>
  </action>
  <action name="action2KFlat">
   <property name="text">
    <string>DCP 2K Flat ( 1998x1080 - 1.85:1 )</string>
   </property>
  </action>
  <action name="action2KScope">
   <property name="text">
    <string>DCP 2K Scope ( 2048x858 - 2.39:1 ) </string>
   </property>
  </action>
  <action name="action4KFlat">
   <property name="text">
    <string>DCP 4K Flat ( 3996x2160 - 1.85:1 )</string>
   </property>
  </action>
  <action name="action4KScope">
   <property name="text">
    <string>DCP 4K Scope ( 4096x1716 - 2.39:1 )</string>
   </property>
  </action>
  <action name="actionDSM4K">
   <property name="text">
    <string>DSM/DCP Full 4K ( 4096x2160 - 1.9:1 )</string>
   </property>
  </action>
  <action name="actionDSM2K">
   <property name="text">
    <string>DSM/DCP Full 2K ( 2048x1080 - 1.9:1 )</string>
   </property>
  </action>
  <action name="actionAdaptRatio">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Adapt to current ratio</string>
   </property>
  </action>
  <action name="action4KUHD">
   <property name="text">
    <string>4K UHD ( 3840x2160 - 1.78:1 )</string>
   </property>
  </action>
  <action name="action8KUHD">
   <property name="text">
    <string>8K UHD ( 7680x4320 - 1.78:1 )</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataComboBox</class>
   <extends>QComboBox</extends>
   <header>UI/datacombobox.h</header>
  </customwidget>
  <customwidget>
   <class>AutoSelectSpinBox</class>
   <extends>QSpinBox</extends>
   <header>duqf-widgets/autoselectspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
